Client/src/components/common/date-picker/index.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "./style.css";
+const days = ["Sat-Mon-Wedn", "Sun-Tues-Thurs"];
```

Comment:
remove the days from here and add them by props to the component. 
that will make your component more reusable